docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -65,6 +80,145 @@ As WordPress loads, it will first check for a custom template override in your t
+  `var product_id = $('.bc-product__pricing').attr('data-product-price-id')`
+
+Generally the WordPress plugin works with post IDs, not product IDs, so the latter is rarely needed on the client side.
+
+To retrieve additional information about the product in the browser, there is a REST API endpoint. It's primary purpose is supporting the product block interface in the WordPress admin, but it can be used anywhere to retrieve a small subset of the information about the product. The endpoint is `/wp-json/bigcommerce/v1/products`.
+
+### Variants
+Retrieve a Product object as above. From there, you can retrieve information about variants.
+
+`$variants = $product->variants;`
+
+The returned objects will match the schema from the BigCommerce API.
+
+```php
+$variant_ids = wp_list_pluck( $variants, 'id' );
+foreach ( $variants as $variant ) {
+  $sku = $variant->sku;
+  $inventory = $variant->inventory_level;
+}
+```
+
+On the client side, variant details are available in the product form.
+
+```php
+var variants = JSON.parse($('[data-js="product-variants-object"]').attr('data-variants'));
+var variant_ids = variants.map( variant => variant.variant_id );
+```
+
+The schema does not completely match the API data. It has been adjusted to suit the needs of the product form. Properties include:
+- variant_id
+- price
+- formatted_price
+- sku
+- disabled
+- disabled_message
+- image
+- option_ids
+
+### Channels
+The current channel is available through a Connections object.
+
+```php
+$connections = new \BigCommerce\Taxonomies\Channel\Connections();
+$channel     = $connections->current();
+```
+
+The response is a `WP_Term` object, with meta containing the channel ID:
+
+```php
+$channel_name = $channel->name;
+$channel_id   = get_term_meta( $channel->term_id, \BigCommerce\Taxonomies\Channel\Channel::CHANNEL_ID, true );
+```
+
+The channel ID is not available anywhere on the client side.
+### Customers
+
+A logged out user does not have any customer information. For a logged in user, you can create a Customer object and get information about that customer.
+
+```php
+$customer    = new \BigCommerce\Accounts\Customer( get_current_user_id() );
+$customer_id = $customer->get_customer_id();
+```
+
+Aside from the ID, no customer information is cached in WordPress. Retrieving additional information will make an API call.
+
+```php
+$profile   = $customer->get_profile();
+$addresses = $customer->get_addresses();
+$orders    = $customer->get_orders( $page, $limit );
+$order     = $customer->get_order_details( $order_id );
+```
+
+
+The customer ID is not available anywhere on the client side.
+
+### Customer Groups
+Similar to the customer ID, the customer group ID is available via the Customer object.
+
+```php
+$customer = new \BigCommerce\Accounts\Customer( get_current_user_id() );
+$group_id = $customer->get_group_id();
+```
+
+Additional information about the customer group is not cached in WordPress, but can be requested from the BigCommerce API:

```

Comment:
Additional information about the customer group is not cached in WordPress. **However, you can request more information from the BigCommerce API:**